Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -479,12 +517,18 @@ private static void WriteAutoSwapOngoing()
             }
         }
 
-        // Throws on failure
-        private static async Task PostAsync(string path, string requestId, string content = null)
+        private static async Task PostKuduAsync(string path, string requestId, string content = null)
```

Comment:
I would have an overload taking host name (then you can pass scm or main site hostname).   We should preserve the ipaddess semantic since it was there to solve DNS issue.